psi4/src/export_mints.cc

```diff
@@ -1058,7 +1058,7 @@ void export_mints(py::module& m)
         .def("irrep_labels",
              [](Molecule& mol) {
                  std::vector<std::string> ret;
-                 char** labels = mol.irrep_labels();
+                 std::vector<std::string> labels = mol.irrep_labels();
                  int nirrep = mol.point_group()->char_table().nirrep();
```

Comment:
Actually  since `mol.irrep_labels()` returns `std::vector<std::string>` then the whole lambda can go now. The function should be bound directly without the need for the wrapper.